src/light.ts

```diff
@@ -0,0 +1,252 @@
+// use package in production
+// const uc = require("uc-integration-api");
+//uc.init("light-driver.json");
+
+import uc from "integration-api";
+import Button from "integration-api/dist/lib/entities/button.js";
+import Light from "integration-api/dist/lib/entities/light.js";
+import MediaPlayer, {
+  ATTRIBUTES,
+  STATES,
+} from "integration-api/dist/lib/entities/media_player.js";
+import {
+  FEATURES as MEDIAPLAYER_FEATURES,
+  ATTRIBUTES as MEDIAPLAYER_ATTRIBUTES,
+  STATES as MEDIAPLAYER_STATES,
+  DEVICECLASSES as MEDIAPLAYER_DEVICECLASSES,
+} from "integration-api/dist/lib/entities/media_player.js";
+import { COMMANDS as BUTTONCOMMANDS } from "integration-api/dist/lib/entities/button.js";
+import { STATUS_CODES } from "http";
+import {
+  DEVICE_STATES,
+  EVENTS as API_EVENTS,
+} from "integration-api/dist/lib/api_definitions.js";
+import { CommandHandler } from "integration-api/dist/lib/entities/entity.js";
+import {
+  COMMANDS as LIGHT_COMMANDS,
+  STATES as LIGHT_STATES,
+  ATTRIBUTES as LIGHT_ATTRIBUTES,
+  FEATURES as LIGHT_FEATURES,
+} from "integration-api/dist/lib/entities/light.js";
```

Comment:
Either the export(s) of the integration library needs to be refactored, or the import needs to be handled differently.
- The client should only be required to have a single import.
- Internal pathes of the integration library should be private: for example the `dist` subpath is an internal detail.